### PR TITLE
Fix ort results scanner interface

### PR DESCRIPTION
**Pull request by awischnewski**
Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

Improves the issues reported in #20 - happy to receive feedback and change as required.

### CODE REVIEW

**Comment by tsteenbe** [comment]
Hi @awischnewski  thank you for your contribution, just back from my holidays so still catching up on things.

Will test your PR this later this week but my initial feedback would be to improve the commit messages.
Propose the below as a suggestion.

Change "make disable variables for advisor, evaluater and scanner false by default to better match with description" to

  ```
Set default value for Advisor, Evaluator and Scanner disable variables

Signed-off-by: Alexander Wischnewski <alexander.wischnewski@driveblocks.ai>
 ```

Change "fix ORT_RESULTS_SCANNER_FILE to use scan-result correctly instead of analyzer-result"
 
 ```
 Fix up incorrect value for ORT_RESULTS_SCANNER_FILE 
 
 Variable incorrectly points to analyzer-result instead of scan-result which is default output file name per [1].

[1]: https://github.com/oss-review-toolkit/ort/blob/7aadaf868bb82292e86a2c2681e4c0f4da4269f0/cli/src/main/kotlin/commands/ScannerCommand.kt#L220

Signed-off-by: Alexander Wischnewski <alexander.wischnewski@driveblocks.ai>
 ```

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -75,13 +75,13 @@ variables:
     description: |
       The Git revision of the ORT configuration repository to use.
   ORT_DISABLE_ADVISOR:
-    value: ""
```
Comment (tsteenbe) [inline .gitlab-ci.yml:78]: The ORT_DISABLE_ADVISOR, ORT_DISABLE_EVALUATOR and ORT_DISABLE_SCANNER are intentionally left empty else you can not set them from your CI/CD variables for your jobs. Default values are set in https://github.com/oss-review-toolkit/ort-gitlab-ci/blob/main/scripts/set-defaults.sh#L9-L11.

I disable the Advisor as ourdefault advisory provider is OSS Index (see [.gitlab-ci.yml#L128](https://github.com/oss-review-toolkit/ort-gitlab-ci/blob/c17059115e81e29a17a91c3b79f48b593526f8b7/.gitlab-ci.yml#L128)) which requires the ORT user to accept their terms and conditions. Let me shoot in a PR to change our default to OSV and then we can enable the advisor by default.

ORT_DISABLE_SCANNER was set to true by default as not every ORT user might want to scan the sources and scanning can take a long time which is not good for a first time user  experience.

Comment (tsteenbe) [reply .gitlab-ci.yml:78]: Created the PR to change to use OSV by and enable the advisor by default, see https://github.com/oss-review-toolkit/ort-gitlab-ci/pull/22

**Comment by sschuberth** [comment]
Closing due to conflicts with https://github.com/oss-review-toolkit/ort-gitlab-ci/pull/26 that was merged meanwhile. We anyway expect this project to be deprecated / archived soon in favor of a new GitLab integration that shares some base with https://github.com/oss-review-toolkit/ort-ci-github-action.

**Pull request closed without merging**